scripts/workspace.py

```diff
@@ -15,33 +16,21 @@
 # specific language governing permissions and limitations
 # under the License.
 
-name: Typos Check
+import subprocess
+import sys
+from constants import PACKAGES
 
-on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
 
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
-  cancel-in-progress: true
+if __name__ == "__main__":
```

Comment:
Contributors should know how to use this script?

Reply:
I've created workspace.py for personal use. I'm hesitant to share this with contributors. Ideally, contributors should focus on specific components such as core or python binding. It's unlikely that they'll need to operate across all components.